kubestash/__init__.py

```diff
@@ -584,27 +594,35 @@ def get_stream_client(args):
         sys.exit(1)
 
     # take the first stream we find... not sure if there are any caveats in doing this.
-    arn = response['Streams'][0]['StreamArn']
+    if len(response['Streams']) > 0:
+        arn = response['Streams'][0]['StreamArn']
 
-    if args.verbose:
-        print("using DynamoDB Stream ARN: {arn}".format(arn=arn))
+        if args.verbose:
+            print("using DynamoDB Stream ARN: {arn}".format(arn=arn))
 
-    response = client.describe_stream(StreamArn=arn, Limit=100)
+        response = client.describe_stream(StreamArn=arn, Limit=100)
 
-    if not response['StreamDescription']['Shards']:
-        print("fatal: no shards found for DynamoDB stream '{arn}'.".format(arn=arn))
-        sys.exit(1)
+        if not response['StreamDescription']:
+            print("fatal: no StreamDescription found for DynamoDB stream '{arn}'.".format(arn=arn))
+            sys.exit(1)
 
-    shard_id = response['StreamDescription']['Shards'][0]['ShardId']
+        if not response['StreamDescription']['Shards']:
+            print("fatal: no shards found for DynamoDB stream '{arn}'.".format(arn=arn))
+            sys.exit(1)
+        try:
```

Comment:
Wondering if the pod will restart properly after this. (What do we keep as a healthcheck?)